### PR TITLE
Document production BigQuery data layout

### PR DESCRIPTION
**Pull request by jklukas**
This PR adds a link to the evolving [Data Access Continuity Guide](https://docs.google.com/document/d/1nlzhRGGwAaClwbotd0oWnnkB5GcvpodIxN3Dk5vWvNI/edit#) and documents the layout and naming of datasets and tables in the reconfigured `shared-prod` GCP project.

What is written here is not yet strictly true. Merging this PR will have to wait on full promotion of the GCP ingestion pipeline to prod, which will hopefully happen next week. It also depends on moving derived tables to shared-prod, which will also hopefully happen next week.

Viva Doc Days!

### CODE REVIEW

**Comment by jklukas** [comment]
Also see [draft email to be sent to fx-data-dev](https://docs.google.com/document/d/1WsmnZTIKAGiJ7OL6SiZ_nR18NI08VFrczcxdQjk12p0/edit):

File: src/cookbooks/bigquery.md [hunk 1]
```diff
@@ -163,37 +166,84 @@ Note: this is very similar to [API Access](bigquery.md#gcp-bigquery-api-access),
...
+|   |`payload_bytes_raw` |Raw JSON payloads as received from clients, used for reprocessing scenarios, a.k.a. "landfill" (_restricted_) |
+|   |`payload_bytes_decoded` |`gzip`-compressed decoded JSON payloads, used for reprocessing scenarios  |
+|   |`payload_bytes_error` |`gzip`-compressed JSON payloads that were rejected in some phase of the pipeline; particularly useful for investigating schema validation errors|
 |   |`search`|Search data imported form parquet (_restricted_)|
-|   |`static`|Static data for use with analysis|
-|   |`telemetry`|User-facing views on imported parquet data and tables generated from [BigQuery ETL](https://github.com/mozilla/bigquery-etl)     |
-|   |`telemetry_raw`|Imported parquet data|
+|   |`static`|Static tables, often useful for data-enriching joins|
 |   |`tmp`|Temporary staging area for parquet data loads|
+|   |`udf` |Persistent user-defined functions defined in SQL|
```
Comment (tdsmith) [inline src/cookbooks/bigquery.md]: Can this link to https://github.com/mozilla/bigquery-etl/tree/master/udf or a UDF documentation page?

File: src/cookbooks/bigquery.md [hunk 2]
```diff
@@ -163,37 +166,84 @@ Note: this is very similar to [API Access](bigquery.md#gcp-bigquery-api-access),
...
+|   |`udf_js` |Persistent user-defined functions defined in JavaScript|
```
Comment (tdsmith) [inline src/cookbooks/bigquery.md]: Can this link to https://github.com/mozilla/bigquery-etl/tree/master/udf_js or a UDF documentation page?

Comment (tdsmith) [reply src/cookbooks/bigquery.md]: I opened #312 for the long term solution but I think these links would be useful in the interim.

File: src/cookbooks/bigquery.md (hunk 1 above)
Comment (jklukas) [reply src/cookbooks/bigquery.md]: Will do

**Comment by jklukas** [comment]
@tdsmith Thank you very much for reading and commenting. I've updated to provide additional context on UDFs and to link to the source as you suggested.

File: src/cookbooks/bigquery.md [hunk 3]
```diff
@@ -163,37 +166,85 @@ Note: this is very similar to [API Access](bigquery.md#gcp-bigquery-api-access),
...
+- _User-facing (unsuffixed) datasets_ (`telemetry`, `activity_stream`, etc.) contain user-facing views on top of the tables in the corresponding stable and derived datasets.
+
+The table and view types referenced above are defined as follows:
+
+- _Live ping tables_ are the final destination for the telemetry ingestion pipeline. Dataflow jobs process incoming ping payloads from clients, batch them together by document type, and load the results to these tables approximately every five minutes, although a few document types are opted in to a more expensive streaming path that makes records available in BigQuery within seconds of ingestion. These tables are partitioned by date according to `submission_timestamp` and are also clustered on that same field, so it is possible to make efficient queries over short windows of recent data such as the last hour. They have a rolling expiration period of 30 days, but that window may be shortened in the future. Analyses should only use these tables if they need results for the current (partial) day.
```
Comment (mreid-moz) [inline src/cookbooks/bigquery.md]:
```suggestion
- _Live ping tables_ are the final destination for the [telemetry ingestion pipeline](https://mozilla.github.io/gcp-ingestion/). Dataflow jobs process incoming ping payloads from clients, batch them together by document type, and load the results to these tables approximately every five minutes, although a few document types are opted in to a more expensive streaming path that makes records available in BigQuery within seconds of ingestion. These tables are partitioned by date according to `submission_timestamp` and are also clustered on that same field, so it is possible to make efficient queries over short windows of recent data such as the last hour. They have a rolling expiration period of 30 days, but that window may be shortened in the future. Analyses should only use these tables if they need results for the current (partial) day.
```

File: src/datasets/ping_intro.md [hunk 4]
```diff
@@ -127,8 +127,11 @@ Try to limit the size of your data by controlling the date range, etc.
 
 #### Accessing the Data
 
-You can access raw ping data from an [ATMO cluster][atmo] using the [Dataset API][dataset].
-Raw ping data are not available in [re:dash][stmo].
+Ping data lives in BigQuery and is accessible in [re:dash][stmo];
+see our [BigQuery intro](../cookbooks/bigquery.md).
+There is currently limited history for main pings available in BigQuery;
+an import of historical data is planned, but without a determined timeline,
```
Comment (mreid-moz) [inline src/datasets/ping_intro.md]:
```suggestion
an import of historical data is [planned](https://bugzilla.mozilla.org/show_bug.cgi?id=1568042), but without a determined timeline,
```

**Pull request merged**